numpy/_core/_add_newdocs.py

```diff
@@ -943,12 +943,12 @@
     dtype : data-type, optional
         By default, the data-type is inferred from the input data.
     order : {'C', 'F', 'A', 'K'}, optional
-        Memory layout.  'A' and 'K' depend on the order of input array a.
-        'C' row-major (C-style),
-        'F' column-major (Fortran-style) memory representation.
-        'A' (any) means 'F' if `a` is Fortran contiguous, 'C' otherwise
-        'K' (keep) preserve input order
-        Defaults to 'K'.
+        The memory layout of the output ndarray. The output of 'A' and 'K' depend on the memory layout of input a.
+        'C' gives a row-major layout (C-style),
+        'F' gives a column-major layout (Fortran-style).
+        Both 'C' and 'F' ensure a contiguous result.
+        'A' (any) is equivalent to 'F' if input a is non-contiguous or Fortran-contiguous, otherwise, it is equivalent to 'C'. If the desired output is a non-contiguous Fortran-style ndarray, the user should set ``copy`` to ``True`` to ensure this.
```

Comment:
Can we please just add a minimal thing that `'A'`, like `'K'`, will not copy to ensure a contiguous output?
Yeah, maybe still confusing, so mentioning the opposite that 'C' and 'F' will copy if needed is maybe clarifying.